src/content/reducers/profile-view.js

```diff
@@ -347,39 +353,39 @@ export const selectorsForThread = (threadIndex: ThreadIndex): SelectorsForThread
       getCallTreeFilters,
       CallTreeFilters.getCallTreeFilterLabels
     );
-    const getRangeFilteredThread: State => Thread = createSelector(
+    const getRangeFilteredThread = createSelector(
       getThread,
       getDisplayRange,
-      (thread: Thread, range: StartEndRange) => {
+      (thread: Thread, range: StartEndRange): Thread => {
```

Comment:
So this is the change I did everywhere below. Flow+reselect didn't like how the function was typed, and we had errors for `createSelector`s below with a function parameter with such a type.

Instead, I type the result of the "reduce" function which implicitely types the selector.

TBH I'm not sure what happens with the previous way of writing types. The error was something like: "expected object instead of static typed function". Here is how `createSelector` is typed: https://github.com/flowtype/flow-typed/blob/master/definitions/npm/reselect_v2.x.x/flow_v0.37.x-/reselect_v2.x.x.js#L1. I don't really understand the syntax there so...

Anyway I prefer the new way of typing :)